lib/jsdom/living/nodes/Document-impl.js

```diff
@@ -607,6 +608,28 @@ class DocumentImpl extends NodeImpl {
     });
   }
 
+  // https://dom.spec.whatwg.org/#dom-document-createcdatasection
+  createCDATASection(data) {
+    if (data.indexOf("[[>") !== -1) {
```

Comment:
No worries about the oversight--thanks for doing the work of putting together the PR.

The tilde with `indexOf` is convenient and fine to my taste...

There is of course always the question of whether a feature--even if a core part of the language--is well-used enough to avoid being too obscure for the average developer reading the code.

It is, I think, similar to the challenge with [protologisms](https://en.wikipedia.org/wiki/Protologism)--the novel use may end up bringing people along to adopt a more efficient communication standard or it may puzzle them or both. :)